app/serializers/book_serializer.rb

```diff
@@ -1,4 +1,3 @@
 class BookSerializer < ActiveModel::Serializer
-  attributes :id, :gender, :author, :image, :title, :publisher, :year
-  has_many :rents
+  attributes :id, :author, :title, :image, :publisher, :year, :gender
```

Comment:
If  you remove this relation in the serializer, you can remove the `includes` in the controller

Reply:
Done